filters/resources.txt

```diff
@@ -2344,6 +2344,14 @@ twitch-videoad.js application/javascript
 			return ourMediaPlayer;
 		}
 	});
+	var realFetch = window.fetch;
+	window.fetch = function() {		
+		if (JSON.stringify(arguments).includes("access_token"))
```

Comment:
Maybe a little more robust and faster:
`if (typeof arguments[0] === "string" && arguments[0].includes("/access_token"))`

Reply:
Thank god people like you exist, helping brainlet fucks like me <3